react-native-app/views/Confirmation.tsx

```diff
@@ -32,31 +36,37 @@ const Confirmation : React.FC = ({route}) => {
         setError('You must enter confirmation code');
       }
     };
+
+    const resendConfirmationCode = async () => {
+      await Auth.resendSignUp(
+        email
+      )
+    }
   
   return (
     <View style={styles.container}>
-      <Text>Check your email for the confirmation code.</Text>
-      <Input
-        value={authCode}
-        placeholder="123456"
-        onChange={(text : any) => setAuthCode(text)}
-      />
-      <Button onPress={() => confirmSignUp()}>Confirm Sign Up</Button>
-      <Text>{error}</Text>
+      <View style={styles.box}>
+        <Text style={styles.text}>Check your email for the confirmation code.</Text>
+         <Input
+            value={authCode}
+            placeholder="123456"
+            onChange={(text : any) => setAuthCode(text)}
+          />
+        <Text style={styles.text}>{error}</Text>
+        <View style={styles.buttons}>
+          <Button  onPress={() => resendConfirmationCode()}  backgroundColor={'red'}>Resend Code</Button>
```

Comment:
Make sure this background color 'red' is the same red we use in the rest of the app. If it is cool, if not then use the `Colors.red`